ui/yarn.lock

```diff
@@ -6076,7 +6076,7 @@ fb-watchman@^2.0.0:
   dependencies:
     bser "^2.0.0"
 
-fbjs@^0.8.0, fbjs@^0.8.1, fbjs@^0.8.15, fbjs@^0.8.16:
+fbjs@^0.8.0, fbjs@^0.8.15, fbjs@^0.8.16:
```

Comment:
why package.json unchanged, yarn.lock changed?

Reply:
discarded the change